.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,151 @@
+name: Integration Tests
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    name: Build Tests
```

Comment:
So, they wont run because I excluded the `ci/*` folder from the root Cargo.toml, because I didn't want to add the annotations to ignore the wasm target architecture (otherwise `cargo release` will break when it tries to compile the integration tests). Also it kinda makes sense that the integration tests are separate from the rest of the workspace. 

But needing to do this is annoying I agree. I will just add it back into the workspace and add the wasm arch release ignore annotations. We do this for the unit tests package, so I can just make the integration tests a package and keep everything uniform.

Reply:
Now I also have to add a way to only run the integration / unit tests too, but that shouldnt be too bad.

Reply:
Actually, Im going to leave `ci` folder outside of the workspace as long as you dont have any complaints about that, because I forgot I would also have to make the entire root workspace use nightly as well. I dont want to do that just for the integration tests.